packages/main/src/plugin/kubernetes/contexts-manager.ts

```diff
@@ -917,6 +927,9 @@ export class ContextsManager {
         return;
       }
       options.timer = setTimeout(() => {
+        if (options.cancel) {
```

Comment:
From my understanding this do not cancel the setTimeout, it just set a `cancel` property, and wait for it to be triggered, then the first line are to check if it has been cancelled.

Reply:
Yes, right, this is what is does. But you are right cancelling the timer directly would be simpler. I'll change

Reply:
Okey thanks